File: class/pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do 
    {
    dollars = get_float("Change owed : ");
    }
    while (dollars < 0);
    
    int cents = round(dollars * 100);
    
    int q = (float)cents / 25.0;
    int quarter = cents - 25*q;
    int d = (float)quarter / 10.0;
    int dime = quarter - 10*d;
    int n = (float)dime / 5.0;
    int nickel = dime - 5*n;
    int p = (float)nickel / 1.0;
    int penny = nickel - 1*p;
    
    int coins = q + d + n + p;
    printf("%d\n", coins);
}

File: class/pset6/cash.py
import cs50
import math

dollars = cs50.get_float("Change owed: ")
while dollars < 0:
    dollars = cs50.get_int("Change owed: ")
cents = round(dollars * 100)
q = (int)(cents / 25.0)
quater = cents - 25.0*q
d = (int)(quater / 10.0)
dime = quater - 10.0*d
n = (int)(dime / 5.0)
nickel = dime - 5.0*n
p = (int)(nickel / 1.0)
penny = nickel - 1.0*p

coins = q + d + n + p
print(f"{coins}")
